src/plugins/intel_gna/model_subrequest.hpp

```diff
@@ -0,0 +1,47 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <cstdint>
+#include <functional>
+
+enum class GNARequestWaitStatus;
+
+namespace GNAPluginNS {
+
+class ModelSubrequest {
+public:
+    enum class Status { kReady, kOngoing, kCompleted, kAborted };
```

Comment:
consider to use/share `GNARequestWaitStatus`

Reply:
done